src/main/java/com/minecolonies/coremod/colony/buildings/BuildingMiner.java

```diff
@@ -120,28 +129,29 @@
     /**
      * Max depth the miner reaches at level 3.
      */
-    private static final int MAX_DEPTH_LEVEL_3 = 5;
+    private static final int                       MAX_DEPTH_LEVEL_3  = 5;
     /**
      * Stores the levels of the miners mine. This could be a map with (depth,level).
      */
     @NotNull
-    private final List<Level> levels      = new ArrayList<>();
+    private final        List<Level>               levels             = new ArrayList<>();
+    private final        Map<ItemStorage, Integer> keepX              = new HashMap<>();
     /**
      * True if shaft is at bottom limit.
      */
-    public  boolean clearedShaft       = false;
+    public               boolean                   clearedShaft       = false;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make clearedShaft a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)